### PR TITLE
BAEL-6194 Retrieve the Value of an HTML Input Using Selenium WebDriver in Java

### PR DESCRIPTION
**Pull request by Michaelin007**

### CODE REVIEW

File: testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    private static final String URL = "https://duckduckgo.com/";
+    private static final String INPUT_ID = "search_form_input_homepage";
+
+    @BeforeEach
+    public void setUp() {
+
+        WebDriverManager.chromedriver().setup();
+        driver = new ChromeDriver();
+
```
Comment (KevinGilmore) [inline testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java:26]: remove the empty lines as per guidelines (no blank lines before/after curly braces)

File: testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    @AfterEach
+    public void tearDown() {
+        driver.quit();
+    }
+
+    @Test
+    public void givenDuckDuckGoHomePage_whenInputHelloWorld_thenInputValueIsHelloWorld() {
+
```
Comment (KevinGilmore) [inline testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java:36]: Let's remove this one, too.

**Review by KevinGilmore — COMMENTED** [review]
Blank lines

File: testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java (hunk 1 above)
Comment (Michaelin007) [reply testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java]: thanks, @KevinGilmore 
Fixed

File: testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java (hunk 2 above)
Comment (Michaelin007) [reply testing-modules/selenium-junit-testng/src/test/java/com/baeldung/selenium/webdriver/SeleniumWebDriverUnitTest.java]: done

**Pull request merged**